mne/viz/tests/test_utils.py

```diff
@@ -161,4 +161,12 @@ def test_validate_if_list_of_axes():
     plt.close('all')
 
 
+def test_center_cmap():
+    """Test centering of colormap."""
+    import matplotlib.cm as cm
+    from matplotlib.colors import LinearSegmentedColormap
+    cmap = center_cmap(cm.get_cmap("RdBu"), -5, 10)
+    assert isinstance(cmap, LinearSegmentedColormap)
+
```

Comment:
I would add an actual test for the behavior.

```py
    from matplotlib.pyplot import Normalize
    from numpy.testing import assert_allclose
    new_colors = cmap(Normalize(-5, 10)([-5, 0, 10]))
    reference = cm.RdBu([0., 0.5, 1.])
    assert_allclose(new_colors,  reference)
```

Reply:
and maybe `assert not assert_almost_equal(cmap([0.5]), reference[1])`

Reply:
Done!